wallet/wallet.go

```diff
@@ -190,17 +194,24 @@ type (
 )
 
 type Balance struct {
-	TotalAmount     types.Amount // 总余额
-	UnspentAmount   types.Amount // 可用余额
-	UnConfirmAmount types.Amount // 待确认
-	SpendAmount     types.Amount // 已花费
+	TotalAmount       Amount // 总余额
+	UnspentAmount     Amount // 可用余额
+	LockAmount        Amount // 锁定
+	UnconfirmedAmount Amount // 待确认
+	SpendAmount       Amount // 已花费
+}
+
+type Amount struct {
+	Coin   string
+	Value  int64
+	FValue float64
```

Comment:
What's the `FValue`? 
and  looks a lot of boilerplate code here.
```golang 
	(&types.Amount{Value: usableAmount.Value, Id: id}).ToCoin()
```
and  code like
```golang
	balance.LockAmount.Value += addr.balanceMap[id].LockAmount.Value
	balance.SpendAmount.Value += addr.balanceMap[id].SpendAmount.Value
	balance.TotalAmount.Value += addr.balanceMap[id].TotalAmount.Value
	balance.UnconfirmedAmount.Value += addr.balanceMap[id].UnconfirmedAmount.Value
```
I think you should reconsider the design of the new `Amount` data structure
```golang
type Amount struct {
	Coin   string
	Value  int64
	FValue float64
```

Reply:
ok